crates/ruff_linter/resources/test/fixtures/pycodestyle/E20.py

```diff
@@ -75,6 +75,21 @@
     x, y = y, x
 a[b1, :] == a[b1, ...]
 b = a[:, b1]
+
+#: E203 linebreak before ]
+predictions = predictions[
+    len(past_covariates) // datamodule.hparams["downsample"] :
+]
+
+#: E203 multi whitespace before :
+predictions = predictions[
+    len(past_covariates) // datamodule.hparams["downsample"]  :
+]
+
+#: E203 tab before :
+predictions = predictions[
+    len(past_covariates) // datamodule.hparams["downsample"]	:
+]
```

Comment:
I added the comment case, but can't figure out how to do the line continuation, without basically rewriting the rule since the \ doesn't show up in the token stream. I could try to adapt E502 for it, if you consider the case important.

On the bright side, both ruff format and black remove a \ in the example situation anyway so there would be no continuing conflict.

Reply:
You could try to use `SimpleTokenizer` when you found a potential violation to lex the text coming right after the token and see if the next token is a line continuation.